sydent/replication/peer.py

```diff
@@ -89,15 +89,21 @@ def __init__(self, sydent, server_name, pubkeys):
         pubkey = self.pubkeys[SIGNING_KEY_ALGORITHM]
         try:
             # Check for hex encoding
-            pubkey_decoded = int(pubkey, 16)
-            logger.warn("Peer %s public key is hex encoded. Please update to base64 encoding", server_name)
-        except:
+            int(pubkey, 16)
+
+            # Decode hex into bytes
+            # For python3 this line will need to be `bytes.fromhex(pubkey)``
+            pubkey_decoded = pubkey.decode('hex')
```

Comment:
let's not be deliberately introducing things that we know are going to break in python 3. `binascii.unhexlify(pubkey)` does exactly what you want here.